doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -442,140 +407,216 @@ Induction
+      based on the definition found in ``Coq.Program.Equality``.
 
-.. tacv:: dependent induction @ident generalizing {+ @ident}
+   .. seealso:: :tacn:`functional induction`
 
-   This performs dependent induction on the hypothesis :n:`@ident` but first
-   generalizes the goal by the given variables so that they are universally
-   quantified in the goal. This is generally what one wants to do with the
-   variables that are inside some constructors in the induction hypothesis. The
-   other ones need not be further generalized.
+.. tacn:: fix @ident @natural {? with {+ ( @ident {* @simple_binder } {? %{ struct @name %} } : @type ) } }
 
-.. tacv:: dependent destruction @ident
+   .. insertprodn simple_binder simple_binder
 
-   This performs the generalization of the instance :n:`@ident` but uses
-   ``destruct`` instead of induction on the generalized hypothesis. This gives
-   results equivalent to ``inversion`` or ``dependent inversion`` if the
-   hypothesis is dependent.
+   .. prodn::
+      simple_binder ::= @name
+      | ( {+ @name } : @term )
+
+   A primitive tactic that starts a proof by induction. Generally,
+   higher-level tactics such as :tacn:`induction` or :tacn:`elim`
+   are easier to use.
 
-See also the larger example of :tacn:`dependent induction`
-and an explanation of the underlying technique.
+   The :n:`@ident`\s (including the first one before the `with`
+   clause) are the names of
+   the induction hypotheses. :n:`@natural` tells on which
+   premise of the current goal the induction acts, starting from 1,
+   counting both dependent and non-dependent products, but skipping local
+   definitions. The current lemma must be composed of at
+   least :n:`@natural` products.
 
-.. seealso:: :tacn:`functional induction`
+   As in a fix expression, induction hypotheses must be used on
+   structurally smaller arguments. The verification that inductive proof
+   arguments are correct is done only when registering the
+   lemma in the global environment. To know if the use of induction hypotheses
+   is correct during the interactive development of a proof, use
+   the command :cmd:`Guarded`.
+
+   :n:`with {+ ( @ident {* @simple_binder } {? %{ struct @name %} } : @type ) }`
+     Starts a proof by mutual induction. The statements to be proven
+     are :n:`forall @simple_binder__i, @type__i`.
+     The identifiers :n:`@ident` (including the first one before the `with` clause)
+     are the names of the induction hypotheses. The identifiers
+     :n:`@name` (in the `{ struct ... }` clauses) are the respective names of
+     the premises on which the induction
+     is performed in the statements to be proved (if not given, Coq
+     guesses what they are).
+
+.. tacn:: cofix @ident {? with {+ ( @ident {* @simple_binder } : @type ) } }
+
+   Starts a proof by co-induction. The :n:`@ident`\s (including the first one
+   before the `with` clause) are the
+   names of the co-induction hypothesis. As in a cofix expression,
```

Comment:
```suggestion
   names of the co-induction hypotheses. As in a cofix expression,
```

Reply:
Got it.  One of the more obscure ways to form a plural in English.